src/main.rs

```diff
@@ -22,9 +26,11 @@ fn startup(mut commands: Commands, asset_server: Res<AssetServer>) {
     commands
         .spawn(PotreePointCloud {
             mesh,
-            point_size: 0.001,
+            point_size: 0.003,
         })
-        .insert(Transform::from_translation(Vec3::new(3.0, 0.0, 0.0)))
+        .insert(Visibility::default())
+        .insert(ComputedVisibility::default())
+        .insert(Transform::from_translation(Vec3::new(0.0, 0.0, 0.0)))
```

Comment:
This is a `SpatialBundle` that you can insert all at once.

https://docs.rs/bevy/latest/bevy/prelude/struct.SpatialBundle.html